fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1420,808 @@ def draw_act(self, obj, layout, value):
+                else:
+                    bpy.ops.object.sm64_export_geolayout_object(export_obj=obj.name)
+                # write model ID, behavior, and level script load
+                if props.export_script_loads and props.model_id != 0:
+                    print(index)
+                    self.export_model_id(context, props, index)
+                    self.export_script_load(context, props)
+        except Exception as e:
+            # pass on multiple export, throw on singular
+            if not props.export_all_selected:
+                raise Exception(e)
+
+    def execute(self, context):
+        props = context.scene.fast64.sm64.combined_export
+        try:
+            self.verify_context(context, props)
+            actor_objs = self.get_export_objects(context, props)
+
+            print(actor_objs)
+            for index, obj in enumerate(actor_objs):
+                props.context_obj = obj
+                self.execute_col(props, obj)
+                self.execute_gfx(props, context, obj, index)
+                # do not export behaviors with multiple selection
+                if props.export_bhv and props.obj_name_bhv and not props.export_all_selected:
+                    self.export_behavior_script(context, props)
+        except Exception as e:
+            props.context_obj = None
+            raisePluginError(self, e)
+            return {"CANCELLED"}
+
+        props.context_obj = None
+        # you've done it!~
+        self.report({"INFO"}, "Success!")
+        return {"FINISHED"}
+
+
+class SM64_CombinedObjectProperties(bpy.types.PropertyGroup):
+    # callbacks must be defined before they are referenced in props
+    def update_preset_behavior(self, context):
+        def update_or_inherit(new_cmd, index, val):
+            if val == "inherit":
+                new_cmd.ineherit_from_export = True
+            else:
+                setattr(new_cmd, f"arg_{index + 1}", val)
+
+        self.behavior_script.clear()
+        bhv_preset = behaviorPresetContents.get(self.preset_behavior_script)
+        if bhv_preset:
+            for cmd in bhv_preset:
+                new_cmd = self.behavior_script.add()
+                new_cmd.expand = False
+                new_cmd.macro = cmd[0]
+                [update_or_inherit(new_cmd, j, val) for j, val in enumerate(cmd[1])]
+
+    # internal object used to keep track during exports. Updated by export function
+    context_obj: bpy.props.PointerProperty(type=bpy.types.Object)
+
+    export_header_type: bpy.props.EnumProperty(
+        name="Header Export",
+        items=[*enumExportHeaderType, ("Custom", "Custom", "No headers are written")],
+        default="Actor",
+    )
+    # level export header
+    level_name: bpy.props.EnumProperty(items=enumLevelNames, name="Level", default="bob")
+    # actor export header
+    group_name: bpy.props.EnumProperty(name="Group Name", default="group0", items=groups_obj_export)
```

Comment:
How about just Group for the name, since this is an enum and I heavily dislike this kind of repetition personally
![image](https://github.com/Fast-64/fast64/assets/87947656/455b6f3e-8dd2-4255-baf5-b978795f3933)